index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Food lover</title>
+  <link rel="stylesheet" href="./styles/style.css"> 
+  <link rel="stylesheet" href="./styles/index.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+</head>
+<body>
+    <header>
+      <img class="floral-nav-divider" src="assets/floral_white_smaller.png" alt="floral divider">
+      <div class="header-text-content">
```

Comment:
`div`s don't provide meaningful information about the structure of our pages so we should avoid using them elements unless it is purely for styling or no semantic element makes sense. I'd recommend using something like `<section>` to replace the outer `div` here.